vcd/resource_vcd_nsxt_dynamic_security_group_test.go

```diff
@@ -467,7 +468,8 @@ func TestAccVcdNsxtDynamicSecurityGroupVdcGroupCriteriaWithVms(t *testing.T) {
 			},
 			{
 				// VM membership is not immediately updated sometimes therewore we apply the same step to check that VM counts are updated
```

Comment:
Maybe clearer like this?

```suggestion
				// VM membership is not immediately updated by CD, therefore we apply the same step to check that VM counts are updated
```

Reply:
Adjusted, but put `VCD` instead of `CD`